es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    const area = width ** 2;
+    const perimeter = 4 * width;
```

Comment:
Rather than creating `area` & `perimeter` properties directly, you should be utilizing the `super` constructor to create them for you